src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature

```diff
@@ -162,112 +157,109 @@ Feature: Descriptors are a set of mechanisms to support flexible enumerations or
                   """
                     {
                         "codeValue": "                      ",
-                        "description": "                    ",
-                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "namespace": "  ",
                         "shortDescription": "                    "
                     }
                   """
              Then it should respond with 400
               And the response body is
-                  """{
```

Comment:
It turns out that putting the `{` next to `"""` breaks things: the code can't read the JSON document properly.